snippets/c.json

```diff
@@ -582,7 +582,7 @@
   "Group section": {
     "prefix": "dgr",
     "body": [
-      "/*! @name ${1:Useless API.}${2: Details...}",
+      "/*! @name ${1:Useless API.}${2: No more details...}",
```

Comment:
Unless there is a strong reason not to, I'd like to see this reverted back to "Details...".

Reply:
Actually, the main idea behind this change (and the previous ones) is that it should make sense if you decide not to change the placeholder.

If you leave just the `Details...`, it would not be clear what is meant. Like, people could think you were trying to say the details are somewhere else and you were just lazy and tried to express it in one word.  
Maybe that's not the great example, but you get the point? While with the `No more details` it's perfectly cleat what you're trying to say.  
And if you decide to write something in the placeholder, the initial text would be gone anyways.

Ofc you could press esc and add `No more`, but why make annoying unnecessary steps, when there's no reason against `No more details`?

Reply:
Ah ok, I think it's clicking with me now :+1: